sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -28,8 +28,17 @@
 
 
 class CertificateClient(AsyncKeyVaultClientBase):
-    """"CertificateClient defines a high level interface for
-    managing certificates in the specified vault.
+    """A high-level asynchronous interface for managing a vault's certificates.
+
+    :param str vault_endpoint: URL of the vault the client will access
+    :param credential: An object which can provide an access token for the vault, such as a credential from
+        :mod:`azure.identity.aio`
+
+    Keyword arguments
+        - **api_version**: version of the Key Vault API to use. Defaults to the most recent.
```

Comment:
str here?  We're not super consistent in specifying type for keyword args, I know in servicebus we try to specify across the board; but obviously prefer whichever makes you more consistent with your scope.